Client/src/main/java/com/Client/ClientController.java

```diff
@@ -8,15 +8,25 @@
 import org.springframework.web.bind.annotation.*;
 
 @Slf4j
-@RestController("/client")
+@RestController
+@RequestMapping("/client")
 public class ClientController {
 
-  @Autowired private ClientService clientService;
+  @Autowired
+  private ClientService clientService;
+
+
+
+@GetMapping("/isClient")
+public boolean isClient(@RequestParam("id") Long id) {
+  return clientService.isClient(id);
+}
```

Comment:
I don't understand why you added this? what's the purpose of this endpoint exactly?
Also, do not name the endpoints like 'isClient' maybe you can rename it to like '/check-client-exists'